src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Classes/IWindowsSetting.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.PowerToys.Run.Plugin.WindowsSettings.Enumerations;
+
+namespace Microsoft.PowerToys.Run.Plugin.WindowsSettings.Classes
```

Comment:
Why is "Classes" part of the namespace here.

Reply:
We have the same on `.Enumerations` and `.Helpers`.

Maybe we can have this:
```
Namespace .WindowsSettings
+- Class WindowsSettingsList with methods and definition of WindowsSetting.
+- all other classes
 - namespace . WindowsSettings.Helpers with all helper classes.
```

Reply:
"class" removed from namespace